common/src/main/java/com/netflix/conductor/common/run/Workflow.java

```diff
@@ -328,7 +320,9 @@ public void setPriority(int priority) {
      * @return the workflow definition name.
      */
     public String getWorkflowName() {
-        Preconditions.checkNotNull(workflowDefinition, "Workflow definition is null");
+        if (workflowDefinition == null) {
+            throw new IllegalArgumentException("Workflow definition is null");
```

Comment:
`Preconditions.checkNotNull` throws `NullPointerException`, i think we should keep the same here.

Reply:
ahh good point, i missed that